### PR TITLE
Volume 3 Chapter 1

### PR DESCRIPTION
**Pull request by DoumanAsh**

### CODE REVIEW

**Review by Potla1995 — CHANGES_REQUESTED** [review]
Line 755:
> It was miracle after miracle that you were able to defeat onee-sama;
> if the three of us had remained at the capital, you'd have no chance of winning.

I'm now working on the Chapter 1!

File: src/3/world1.md [hunk 1]
```diff
@@ -1,154 +1,155 @@
 ![Cover](./midashi01.jpg)
 
-There was standing a solemn ancient tree.
-Being several hundred years old, this tree had growth so much that it looked like it was piercing heavens itself.
-It had countless branches, it's thick leafs were still fresh and it still emitted wild fragrance.
+A solemn ancient tree was standing upright.
+Being several hundred years old, the tree had grown so much that it seemed to be piercing through the heavens.
+It had countless branches; its thick leafs were still fresh emitted a wild fragrance.
```
Comment (DoumanAsh) [inline src/3/world1.md]: `emitted` -> `, emitting` would be better maybe?

File: src/3/world1.md [hunk 2]
```diff
@@ -1,154 +1,155 @@
...
+He wasn't lazing around.
+Saying so, Kai jumped up from the root on which he was sitting.
 
 "Eh, Rinne? I'm kinda worried if she hasn't gotten lost..."
 
-"Don't worry, when it comes to Rinne, she can quickly..."
+"Don't worry. When it comes to Rinne, wherever I am, she can quickly..."
 
 _TL Note:_ TODO `リンネなら俺がどこにいてもすぐ──` I'm not sure how to re-order sentence to not miss the port about Kay's place
+_ED Note:_ How does the above wording sound?
```
Comment (DoumanAsh) [inline src/3/world1.md]: Sounds good, we'll go with it

File: src/3/world1.md [hunk 3]
```diff
@@ -1,154 +1,155 @@
...
-"So for foreign gods it might be normal to sit down for about a month without eating anything when they have serious discussion.
-Carry on, without sleeping or even a single drop of water."
+"So for the foreign gods it might be normal to sit down for about a month without eating anything when they have serious discussions.
+They tend to carry on, without sleeping or even a single drop of water."
 
 "I cannot imagine."
 
 "Yeah, hearing it made me shiver.
-And we were fighting against race that could calmly talk about something like that."
+And we were fighting against races that could calmly talk about something like that."
```
Comment (DoumanAsh) [inline src/3/world1.md]: `races` -> `the race`

I know it is confusing the way author made foreigns gods, but they are referred as singular race always

File: src/3/world1.md [hunk 4]
```diff
@@ -212,69 +213,69 @@ She was able to clearly recall it.
...
 "It feels so good.
-This water, it is spring water, right?
+This water is spring water, right?
 Cold, but so pretty.
 And there is even small fish swimming around."
 
 "U-umu.
 It is filtered water from within underground after all.
-For every living creature in the forest this is a watering place, with abundance of clear water....
+For every living creature in the forest this is a basin, with an abundance of clear water....
```
Comment (DoumanAsh) [inline src/3/world1.md]: `basin` - please check original word, I think it was more like a watering place as if the place where animals gather to drink and wash

File: src/3/world1.md [hunk 5]
```diff
@@ -410,178 +411,180 @@ My body, which has no unnecessary meat, is in fact the best!"
 
 "What is wrong with being flat!?"
 
-_TL Note:_ Oi, Rinne did you get bulling license on our flat goddess!?
+_TL Note:_ Oi, Rinne did you get bullying license on our flat goddess!?
+_ED Note:_ She's not wrong though... :eyes:
```
Comment (DoumanAsh) [inline src/3/world1.md:413]: Remember to add empty lines to separate sentence by empty line, when it is rendered

File: src/3/world1.md [hunk 6]
```diff
@@ -410,178 +411,180 @@ My body, which has no unnecessary meat, is in fact the best!"
...
+It is a big country in the eastern part of the continent, and at the moment it was under control of the foreign gods.
+Humans, who lost in the war, had to reconstruct ruins into cities and live in hiding, somehow scraping by.
+Each of the 18 human cities is guarded by Resistance soldiers, who serve as defence for the people from the foreign gods invasion.
 
-Headquarters of the Resistance was in 8th ward Cassiopeia.
-It was a site of once large scale factory, and in one of the areas there...
+The headquarters of the Resistance was in 8th ward Cassiopeia.
+It was at the site of what was once a large scale factory, and in one of the areas there...
 
 "Jeanne-dono, are you about to depart...?"
```
Comment (DoumanAsh) [inline src/3/world1.md]: I need to check original and re-phrase it.
There should talking about Jeanne being about to depart

Should be `Jeanne-dono is about to depart...?`

**Review by DoumanAsh — COMMENTED** [review]
Thanks

File: src/3/world1.md (hunk 1 above)
Comment (Potla1995) [reply src/3/world1.md]: You're right! It should be
> It had countless branches; its thick leafs were still fresh, emitting a wild fragrance.

File: src/3/world1.md (hunk 2 above)
Comment (Potla1995) [reply src/3/world1.md]: (y)

File: src/3/world1.md (hunk 3 above)
Comment (Potla1995) [reply src/3/world1.md]: Ah I see. My bad. I thought he was talking about all the other races as well!
"The race" is correct!

File: src/3/world1.md (hunk 4 above)
Comment (Potla1995) [reply src/3/world1.md]: I understood... tried to find a better word for "watering place" but could not find one... 
I wanted to go with the meaning 2b here: https://www.merriam-webster.com/dictionary/basin

File: src/3/world1.md (hunk 5 above)
Comment (Potla1995) [reply src/3/world1.md:413]: ! Forgot! >w< Thanks!

File: src/3/world1.md (hunk 6 above)
Comment (Potla1995) [reply src/3/world1.md]: You're absolutely right! Jeanne-dono is not even present there! xD

**Review by Potla1995 — COMMENTED** [review]
Please check newest commit! :)

**Pull request merged**